Lab1/Snake/snake.js

```diff
@@ -103,8 +103,8 @@ const board = {
      */
     clear : function() {
         /* TODO : remove all elements from the board's DOM element, 'el' */
-        while(board.el.firstChild){
-            board.el.removeChild(board.el.firstChild)
+        while(this.el.firstChild){
+            this.el.removeChild(this.el.firstChild)
```

Comment:
What you have works because `board` is a global binding, but it's best to get in the habit of using `this` in the context of an object's methods.